documentation/assemblies/security/assembly-access-operator.adoc

```diff
@@ -0,0 +1,19 @@
+// This assembly is included in the following assemblies:
+//
+// security/assembly-security.adoc
+
+[id='assembly-access-operator-{context}']
+= Using the Access Operator to manage client connections (early access)
+
+[role="_abstract"]
+The Strimzi Access Operator simplifies how applications connect to a Kafka cluster managed by the Cluster Operator. 
```

Comment:
I would maybe change this a bit to:
* Make it clear that this is _optional_
* Instead of _simplifies how applications connect_ say something like _simplifies sharing of Kafka connection information and credentials between namespaces_